pypesto/optimize/optimizer.py

```diff
@@ -382,12 +383,37 @@ def minimize(
                 'trust-krylov',
                 'trust-constr',
             ]
+            # switch off passing over functions if not applicable (e.g.
+            # NegLogParameterPrior) since grad/hess attributes do not exist
+            if not isinstance(objective, Objective):
+                if not hasattr(objective, 'grad'):
+                    objective.grad = False
+                if not hasattr(objective, 'hess'):
+                    objective.hess = False
+            # Todo Resolve warning by implementing saving of hess temporarily
+            #  in objective and pass to scipy seperately
+            if objective.hess is True:
+                logger.warning(
+                    "scipy.optimize.minimize does not support "
+                    "passing fun and hess as one function. Hence "
+                    "for each evaluation of hess, fun will be "
+                    "evaluated again. This can lead to increased "
+                    "computation times. If possible, separate fun "
+                    "and hess."
+                )
```

Comment:
@MerktSimon This warning is printed several thousand times during an optimization (see e.g. https://github.com/ICB-DCM/pyPESTO/actions/runs/3741447433/jobs/6351066417). I think it would be much preferable to use `warnings.warn`, where this will only be printed once, by default.

Reply:
I agree. Since this PR is already merged now, should I just add this to #988 ?

Reply:
Better create a separate PR.